src/socket.ts

```diff
@@ -0,0 +1,51 @@
+import * as socketIo from 'socket.io';
+import { logger } from './utils/logger/logger';
+import { config } from './config';
+
+export class Socket {
```

Comment:
To my understanding, the name 'Socket' of the class is a bit confusing. 
The class actually inhabits all of the socket connections and will always have only a single instance. 
Perhaps think about changing the name to 'SocketsConnector', 'SocketsHolder', 'SocketHandler', etc...

Reply:
renamed that.